manifests/p/Python/Launcher/3.11.5/Python.Launcher.installer.yaml

```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Python.Launcher
+PackageVersion: 3.11.5
+InstallerType: wix
+UpgradeBehavior: install
+Commands:
+- py
+- python
+- pythonw
+- pyw
+FileExtensions:
+- py
+- pyc
+- pyd
+- pyo
+- pyw
+- pyz
+- pyzw
+ProductCode: '{2FB71770-2C2E-42A3-9136-5101D1E930F4}'
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://www.python.org/ftp/python/3.11.5/win32/launcher.msi
+  InstallerSha256: 29157C56C3AEA29485447868FD03C41067D0BE47AE906586962BDD67C88C9996

```

Comment:
```suggestion
  InstallerSha256: 29157C56C3AEA29485447868FD03C41067D0BE47AE906586962BDD67C88C9996
  InstallerSwitches:
    Custom: ALLUSERS=0
```

Reply:
Yes. In Windows Sandbox, the installation process will finally end up in `machine` scope instead of `user` scope. Adding `InstallerSwitches.Custom` for user scope with `ALLUSERS=0" IS better.